owners/OWNERS.example.yaml

```diff
@@ -58,17 +59,40 @@
 - "*.{js,css}": frontend
 - "package.{,lock.}json": somebodyelse
 
+# Prepending a user or team name with a `?` will prevent this owner from being
+# automatically notified or requested as a reviewer. They may still be requested
```

Comment:
This comment is for line 63 :)

Maybe clarify? "They are still an owner, and may still be requested or tagged manually/

Reply:
Clarified :)